stories/organisms/components/filePicker/FilePicker.stories.mdx

```diff
@@ -97,6 +97,34 @@ It is a combination of the [FileInput](/docs/atoms-fileinput--overview) and the
 
 <ArgsTable story="Overview" />
 
+## Clear all
+
+The `clearAll` property allows the user to add a link to delete all selected files.  
```

Comment:
For me, the term "link" is to be avoided because it could be thought that it acts as a real html `link` and would take the user to another page to delete the selected files. But I'm not sure if my remark is relevant !